.github/workflows/main.yml

```diff
@@ -8,15 +8,15 @@ jobs:
       matrix:
         choiceL: [--disable-silent-rules, --enable-silent-rules]
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Create configure
         run: |
           sudo apt-get update -y
-          sudo apt-get install autoconf automake libtool gcc libfftw3-dev libgimp2.0-dev
+          sudo apt-get install autoconf automake libtool gcc libfftw3-dev libgimp-3.0-dev
```

Comment:
ubuntu does not seem to be ready to build GIMP3 plugins at the time of the GIMP3 release (and even worse for ubuntu-latest), so the CI will fail until we reach such an ubuntu version. Maybe we should disable those jobs for now?

Reply:
True. Even though I notice that debian shows libgimp-3.0-dev, it seems that github is not yet ready - we just have to wait. Same for MSYS2, there is a libgimp 3 dev (only 64bit), but github is not including it yet.
We just have to wait.
I think I read somewhere that we also need to wait for gimptool-3.0 to appear later than gimp-3.0

Reply:
As the other parts of the CI will be OK sooner, I think the best option is to disable the linux jobs until we have the required packages in ubuntu. I can do that after the PR.